docs/quickstart.md

```diff
@@ -1,182 +1,364 @@
+**Skills**: It is helpful if you have some familiarity with using the command line, as well as using a text editor and/or JupyterLab.
 
-`mystmd` has capabilities beyond Jupyter Book, for example exporting to scientific PDF documents, and you can use the two tools together! 💚
+**Tools**: You'll need a code and notebook editor ([VSCode](https://code.visualstudio.com/) is great, and we recommend [Jupyter Lab](https://jupyter.org/install) for notebooks) as well as access to a CLI.
 :::
-::::
-
-To follow along with this quickstart tutorial on your own computer, it is helpful if you have some familiarity with using the command line, as well as using a text editor and/or JupyterLab.
 
 (lookout-for-tutorial-actions)=
 
 > 🛠 Throughout the tutorial, whenever you're supposed to _do_ something you will see a 🛠
 
-## Installing the MyST Markdown CLI 📦
+## Install the MyST Markdown CLI 📦
 
-`mystmd` is a command line interface (CLI) that provides modern tooling for technical writing, reproducible science, and creating scientific & technical websites. To get started, install `mystmd`.
+`mystmd` is a {abbr}`CLI (Command Line Interface)` that provides modern tooling for technical writing, reproducible science, and creating scientific & technical websites. 
+These instructions help you install the CLI.
 
-:::{tip} Prerequisites - Install Node
-:class: dropdown
+### Install `conda` and `conda-forge`
 
-You should have these programs installed:
+The easiest way to install MyST is with [the `conda` package manager](https://conda.io/projects/conda/en/latest/).
+`conda` is a multi-language manager that is useful for users across many data science languages like Python, R, Julia, and JavaScript.
 
-- [Node.js](https://nodejs.org) version **>=18.0.0**
-- [Node Package Manager (npm)](https://docs.npmjs.com/about-npm) version **>=7.0.0**
-- A code and notebook editor ([VSCode](https://code.visualstudio.com/) is great, and we recommend [Jupyter Lab](https://jupyter.org/install) for notebooks)
+The easiest way to install `conda` is with [the `conda-forge` distribution](https://conda-forge.org/download/).
+This includes miniconda and a community-driven package index called `conda-forge`.
 
-If the node ecosystem is new to you, see our getting started guides for [installing node](./installing-prerequisites.md).
-:::
+🛠 Install miniconda from [conda-forge](https://conda-forge.org):
 
-🛠 Install the MyST command line tools:
+1. Go to [the conda-forge latest releases section](https://conda-forge.org/download/).
+2. Download the release for your platform and follow the instructions.
+3. To check that you've installed properly, you should be able to execute the following command to print the version of conda:
 
-(installing-myst-tabs)=
-::::{tab-set}
-:::{tab-item} PyPI
+   ```shell
+   $ conda --version
+   conda 24.7.1
+   ```
 
-🛠 Install `node` (<https://nodejs.org>), see [Installing NodeJS](./installing-prerequisites.md):
+### Install Node and NPM
```

Comment:
Is there a reason that these were removed?

There is now no longer a place where we can see how to install via pip or npm.

Reply:
My reasoning was that we want quickstarts and tutorials to be opinionated, ready and executed linearly, and generally not provide many ways of doing something. Since we have [this page with much more information about installing MyST](https://mystmd.org/guide/installing) I figured we should define the one, simplest way to install it from the quickstart, and reference that page for more information.